test/bdd/fixtures/oathkeeper/rules/resource-server.json

```diff
@@ -0,0 +1,34 @@
+[
```

Comment:
no need for this

Reply:
your hub-kms don't care about edv oathkeeper

Reply:
ok, will fix in follow-up ticket